joomla/joomla-cms
# [4.2] Help Screen

Pull Request for Issue # .

### Summary of Changes
Generated the table of contents displayed in the Joomla help (`administrator/index.php?option=com_admin&view=help`)

This should have been done for 4.1 as well but even though it was marked as a release blocker it was ignored.

The helpTOC is generated by comparing the strings COM_ADMIN_HELP_*** in `administrator\language\en-GB\com_admin.ini` with the help pages available in the wiki for the current version.

I have manually added all the help pages to the 4.2 category on the wiki that match the strings and regenerated the TOC.


### Testing Instructions
Apply pr and go to the help. You will see a full list of help items and NEW to this version are two help items for scheduled tasks

## Important
The list of available pages on the wiki is much greater than this list generated for use inside joomla. Someone might want to review and check to see if any should be added.

To make this easier the full list of 238 possible pages is https://docs.joomla.org/Category:Help_screen_4.0
The list of 91 pages included in joomla is https://docs.joomla.org/Category:Help_screen_4.2

### Adding
1. tag the item on the wiki to be in the category ‘Help screen 4.2’
2. add the required string in `administrator/index.php?option=com_admin&view=help`
3. run the script build\helpTOC.php

### Removing
1. remove the item on the wiki from the category ‘Help screen 4.2’
2. remove the string in `administrator/index.php?option=com_admin&view=help`
3. run the script build\helpTOC.php
